site/app/controllers/DockerInterfaceController.php

```diff
@@ -153,6 +155,39 @@ public function addImage() {
         }
     }
 
+    /**
+     * @Route("/admin/remove_image", methods={"POST"})
+     * @return JsonResponse
+     */
+    public function removeImage() {
+        $user = $this->core->getUser();
+        $user_id = $this->core->getUser()->getId();
+        $jsonFilePath = FileUtils::joinPaths(
+            $this->core->getConfig()->getSubmittyInstallPath(),
+            "config",
+            "autograding_containers.json"
+        );
+        $json = json_decode(file_get_contents($jsonFilePath), true);
+        $key = array_search($_POST['image'], $json[$_POST['capability']]);
+        $Verify = $this->core->getQueries()->removeDockerImageOwner($_POST['image'], $user_id);
+        if ($key !== false && $Verify != false) {
```

Comment:
I think we can check if the key is false before we attempt to perform the query, if the container is not listed in the json file then I don't the query would succeed either so we can avoid trying it